app/models/manageiq/providers/autosde/storage_manager/cloud_volume.rb

```diff
@@ -49,4 +50,15 @@ def raw_update_volume(options = {})
     ems.autosde_client.VolumeApi.volumes_pk_put(ems_ref, update_details)
     EmsRefresh.queue_refresh(ems)
   end
+
+  # ================ safe-delete ================
+  def validate_safe_delete_volume
+    {:available => true, :message => nil}
+  end
```

Comment:
@gregoryb-at-ibm can you remove this since we don't need validate_safe_delete_volume anymore?